src/main/java/life/qbic/projectwizard/control/ExperimentImportController.java

```diff
@@ -987,8 +980,7 @@ private void countExistingOpenbisEntities(String space, String project)
     firstFreeBarcode = "";// TODO cleanup where not needed
     currentProjectSamples = new ArrayList<Sample>();
     if (openbis.projectExists(space, project)) {
-      currentProjectSamples.addAll(openbis
-          .getSamplesWithParentsAndChildrenOfProjectBySearchService("/" + space + "/" + project));
+      currentProjectSamples.addAll(openbis.getSamplesOfProject("/" + space + "/" + project));
     }
     List<Experiment> experiments = openbis.getExperimentsOfProjectByCode(project);
     for (Experiment e : experiments) {
```

Comment:
`e` is also used for exceptions and not very expressive. Since you did not really touch that in this PR this is only NIT